src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitInStorage;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, FruitInStorage> FRUITS = new HashMap<>();
```

Comment:
Consider replacing this map to Map<String, Integer> because key of the map is the name of fruit and the value is amount and as a result your FruitInStorage it's a duplication object with the name and amount